pkg/util/error.go

```diff
@@ -0,0 +1,7 @@
+package util
+
+type Error string
```

Comment:
Why are we adding this? Is `errors.New()` insufficient?

Reply:
Oren recommended this pattern so that errors could be compile-time constants. I have absolutely no opinion on whether we do this and will concede to reviewers. If you want me to remove it, I will.

Reply:
Let's figure out the "why" to get you out of the middle of conflicting feedback.

@shomron Is there benefit in creating our own error class instead of using errors.New()? I get the appeal of a constant.